README.md

```diff
@@ -170,9 +177,11 @@ gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV
 --serviceAccount=${SERVICE_ACCOUNT_EMAIL}"
 ```
 
-Above command will trigger a batch inspection Dataflow pipeline that will process only the CSV files that were present in the demo-data GCS bucket (specify in _filePattern_ parameter) before the pipeline was triggered.
+Any files that are added to the bucket after the pipeline was triggered are not inspected.
+
+##### Inspect only new files
 
-Run following command to trigger the inspection pipeline to process only the new files added to the GCS bucket(files added after the pipeline is triggered):
+To trigger a streaming inspection Dataflow pipeline that processes new CSV files that are added to the `demo-data` bucket (specified in `filePattern` parameter), run the following command:
```

Comment:
Can we mention the full name of the bucket which would be <PROJECT_ID>-demo-data?